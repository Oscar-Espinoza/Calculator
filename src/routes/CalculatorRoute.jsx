import React from 'react';
import Calculator from '../components/Calculator';

export default function CalculatorRoute() {
  return (
    <div className="calculator-view">
      <h2 className="calculator-title">Lets do some math</h2>
      <Calculator />
    </div>
  );
}
