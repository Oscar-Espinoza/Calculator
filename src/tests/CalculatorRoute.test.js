import React from 'react';
import TestRenderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import CalculatorRoute from '../routes/CalculatorRoute';

it('renders correctly', () => {
  const tree = TestRenderer.create(
    <BrowserRouter>
      <CalculatorRoute />
    </BrowserRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
