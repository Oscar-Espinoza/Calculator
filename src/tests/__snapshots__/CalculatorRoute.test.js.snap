// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quote-container"
>
  <p
    className="quote"
  >
    <q>
      Mathematics is the queen of the sciences and number theory is the queen of mathematics.
    </q>
    - Carl Friedrich Gauss
  </p>
</div>
`;
